com.sap.cloud.lm.sl.cf.core/src/main/java/com/sap/cloud/lm/sl/cf/core/cf/clients/ServiceCreator.java

```diff
@@ -54,13 +58,31 @@ public void createService(CloudFoundryOperations client, CloudServiceExtended se
     }
 
     private List<String> computeValidServiceOfferings(CloudFoundryOperations client, List<String> possibleServiceOfferings,
-        List<String> existingServiceOfferings) {
+        String servicePlan, Map<String, List<CloudServicePlan>> existingServiceOfferings) {
         List<String> validServiceOfferings = new ArrayList<String>();
         for (String possibleServiceOffering : possibleServiceOfferings) {
-            if (existingServiceOfferings.contains(possibleServiceOffering)) {
+            for (String existingServiceOffering : existingServiceOfferings.keySet()) {
+                if (!existingServiceOffering.equals(possibleServiceOffering)) {
+                    LOGGER.warn(format("Defined service offering \"{0}\" is not existing", possibleServiceOffering));
```

Comment:
What does "defined" mean here? Probably it is better to be replaced with **Requested** or even just remove it.

Reply:
Done

Reply:
Done